__tests__/app.test.js

```diff
@@ -53,4 +53,34 @@ describe("API's", () => {
         });
     });
   });
+  describe("/api/articles/:article_id", () => {
+    test("GET 200: Accepts a parametric endpoint with and article id and returns that article", () => {
+      return request(app)
+        .get("/api/articles/1")
+        .expect(200)
+        .then(({ body }) => {
+          const { article } = body;
+          expect(article).toMatchObject({
+            author: "butter_bridge",
+            title: "Living in the shadow of a great man",
+            article_id: 1,
+            body: "I find this existence challenging",
+            topic: "mitch",
+            created_at: "2020-07-09T20:11:00.000Z",
+            votes: 100,
+            article_img_url:
+              "https://images.pexels.com/photos/158651/news-newsletter-newspaper-information-158651.jpeg?w=700&h=700",
+          });
+        });
+    });
+    test("GET 404: Responds with the appropriate error when passed the wrong article_id", () => {
+      return request(app)
+        .get("/api/articles/14")
+        .expect(404)
+        .then(({ body }) => {
+          const { msg } = body;
+          expect(msg).toBe("Not Found");
+        });
+    });
```

Comment:
A good 404 test, but we're missing a 400 test for when the user provides an article idea with the wrong datatype, which would trigger a PSQL error
For example `/api/articles/katherine`